packages/sdk/src/grpc/translation.ts

```diff
@@ -2065,15 +2065,17 @@ function trAccountTransactionSummary(
 function tokenEvent(event: GRPC_PLT.TokenEvent): PLT.TokenEvent {
     switch (event.event.oneofKind) {
         case 'transferEvent':
-            return {
+            const transferEvent: PLT.TokenEvent = {
                 tag: PLT.TokenEventType.Transfer,
                 from: tokenHolder(unwrap(event.event.transferEvent.from)),
                 to: tokenHolder(unwrap(event.event.transferEvent.to)),
                 amount: PLT.TokenAmount.fromProto(unwrap(event.event.transferEvent.amount)),
-                memo: event.event.transferEvent.memo
-                    ? PLT.CborMemo.fromProto(unwrap(event.event.transferEvent.memo))
-                    : undefined,
             };
+            if (event.event.transferEvent.memo) {
+                transferEvent.memo = PLT.CborMemo.fromProto(unwrap(event.event.transferEvent.memo));
+            }
+
+            return transferEvent;
```

Comment:
since token events now occur in `governance` and `holder` transaction, line 2098 could be updated to:

throw Error('Failed translating "TokenEvent", encountered undefined value');